.devops/deploy-pipelines.yml

```diff
@@ -336,6 +336,7 @@ stages:
   - stage: "Green_API_UAT_test"
     displayName: 'Running API test on green app'
     dependsOn: Deploy_UAT_Green
+    condition: eq(dependencies.Deploy_UAT_Green.result, 'Succeeded')
```

Comment:
Isn't this redundant with the previous line?

Reply:
Fixed with b379d7460a95d10f31e811f8276a6bf4c7e2a3ab